### ISSUE TITLE
Segfault using intigriti

### ISSUE THREAD

**Issue by lkeersmaekers** [root4loot/rescope#8]
rescope -u intigriti.com/public/project/tomorrowland/tomorrowland --burp -o test.ext

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6de0f3]

goroutine 1 [running]:
github.com/root4loot/rescope/internal/url.BBaas(0xc000098e20, 0x1, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x8, ...)
        /home/user/go/src/github.com/root4loot/rescope/internal/url/url.go:60 +0x613
main.main()
        /home/luser/go/src/github.com/root4loot/rescope/main.go:80 +0x6dc
```

**Comment by root4loot** [comment]
Should now be resolved in 7298621

**Comment by lkeersmaekers** [comment]
Tested and works with `rescope -u https://app.intigriti.com/programs/intigriti/intigriti/detail --burp -o test.ext`

Great tool, thank you very much.
